### ISSUE TITLE
FileChooser shown from modal dialog results in unresponsive UI

### ISSUE THREAD

**Issue by karlvr** [violetlib/vaqua#19]
I have a modal dialog that presents a `JFileChooser` using VAqua. After choosing a file and closing the file chooser dialog, I can interact with my modal dialog, but after I close the modal dialog, my application UI doesn't receive clicks.

This only occurs with VAqua and with a modal dialog, and _only_ if I present the dialog using:

```java
VSheet.showDialog(dialog, null);
```

I am investigating, but posting an issue in case it rings any bells!

**Comment by karlvr** [comment]
OK, I can resolve the issue by not calling `setModal(true)` on my dialog before showing it. Something funny happening in connection with sheets and a dialog that is modal?

**Comment by cbfiddle** [comment]
Apparently so, but I was not aware of it.

Sheets are already modal, so there is no need to enable Java’s modal window support, which causes it to respond to events in a special way.

**Comment by karlvr** [comment]
@cbfiddle yeah, a funny one... do you think we should add a correction of this modal status into the client?

**Comment by cbfiddle** [comment]
If it is possible, then it sounds like a good idea.

**Comment by cbfiddle** [comment]
I think this problem has been fixed. Can you confirm?
